### ISSUE TITLE
Add rename folder

### ISSUE THREAD

**Issue by Krzysztof-Cieslak** [fsprojects/Forge#65]
Adds command renaming folders in project or solution. Command `rename folder` accepting parameters `project` and `solution` should be implemented.

## Expected Behavior
- rename folder 
- modify `.fsporj` or `.sln` file 

## Implementation suggestions
Implementation should be done using `Forge.ProjectSystem` module.
Implementation should be placed in `Forge.ProjectManager` module

**Comment by reidev275** [comment]
Is this already done? https://github.com/fsprojects/Forge/blob/master/src/Forge.Core/ProjectManager.fs#L190

**Comment by takemyoxygen** [comment]
Indeed, there's quite some code already but there are some minor issues with that e.g. passing arguments to `renameDir` in the wrong order. It's also unclear whether one should specify folder names relative to Forge working directory or relative to fsproj file location in case these two directories are different.

**Comment by cloudRoutine** [comment]
It's relative to the fsproj location because that's how the folder is represented in the paths of the files. The folder doesn't necessarily exist physically, a folder that only stores linked files wouldn't have a representation on disk.  Solution folders are entirely virtual and have no inherent correspondence with the folder tree on disk.

**Comment by takemyoxygen** [comment]
I might spend some time on that assuming nobody else has started doing that yet.
